### DISCUSSION TITLE
Python 3.10?

### DISCUSSION THREAD

**Discussion by teodorescuserban** [ckan/ckan#7236] [General]
Hello, I was wondering if you considering supporting python 3.10.

Thank you for your time.

**Comment by amercader** [comment]
Absolutely! What version of CKAN are you trying to run in Python 3.10? Are you finding problems that can be addressed?
I suspect that other than maybe some requirement updates it should be pretty straightforward

**Reply by teodorescuserban** [reply]
2.9.7, but we really want to keep ckan core untouched and add on top of it our things. This way it gets much easier to do a ckan upgrade. Going up to python 3.10 is not a priority for us right now, but I was just wondering when that would be possible. On the side, we would like to base our docker image on Alpine (and the last Alpine with python 3.8 EOL for like a month). I hope this helps.

**Comment by pdelboca** [comment]
As @amercader pointed out, we absolutely want to support Python 3.10.

There was an attempt to do it some months ago but it came with some issues of incompatibility between versions: https://github.com/ckan/ckan/pull/6613

It is good to know that that Alphine dropped support for Python 3.8. I think it is a good point to try to narrow how many Python versions we support in CKAN.

Currently we are in the process of releasing the new 2.10 version, but 3.10 will be definitely in next year's backlog!
